--- templates/extinfo_type_2.tt.old	2019-04-11 18:18:36.513251996 +0200
+++ templates/extinfo_type_2.tt	2019-04-11 19:16:15.733307949 +0200
@@ -349,102 +349,6 @@
                             <td class='dataVal'>[% pi_detail.$peer_key.peer_name %]</td>
                           </tr>
                           [% END %]
-                        </table>
-                      </td>
-                    </tr>
-                  </table>
-                </td>
-              </tr>
-              <tr>
-                <td>
-                  <table border="1" cellspacing="0" cellpadding="0" class="data checkstatus">
-                    <tr>
-                      <td class='stateInfoTable2 checkstatus'>
-                        <table border="0">
-                          <tr>
-                            <td class='dataVar'>Active Checks:</td>
-                            <td class='dataVal'>
-                              [% IF service.checks_enabled %]
-                              <div class='checksENABLED'>&nbsp;&nbsp;ENABLED&nbsp;&nbsp;</div>
-                              [% ELSE %]
-                              <div class='checksDISABLED'>&nbsp;&nbsp;DISABLED&nbsp;&nbsp;</div>
-                              [% END %]
-                            </td>
-                          </tr>
-                          <tr>
-                            <td class='dataVar'>Passive Checks:</td>
-                            <td class='dataVal'>
-                              [% IF service.accept_passive_checks %]
-                              <div class='checksENABLED'>&nbsp;&nbsp;ENABLED&nbsp;&nbsp;</div>
-                              [% ELSE %]
-                              <div class='checksDISABLED'>&nbsp;&nbsp;DISABLED&nbsp;&nbsp;</div>
-                              [% END %]
-                            </td>
-                          </tr>
-                          <tr>
-                            <td class='dataVar'>Obsessing:</td>
-                            <td class='dataVal'>
-                              [% IF service.obsess_over_service %]
-                              <div class='checksENABLED'>&nbsp;&nbsp;ENABLED&nbsp;&nbsp;</div>
-                              [% ELSE %]
-                              <div class='checksDISABLED'>&nbsp;&nbsp;DISABLED&nbsp;&nbsp;</div>
-                              [% END %]
-                            </td>
-                          </tr>
-                          <tr>
-                            <td class='dataVar'>Notifications:</td>
-                            <td class='dataVal'>
-                              [% IF service.notifications_enabled %]
-                              <div class='notificationsENABLED'>&nbsp;&nbsp;ENABLED&nbsp;&nbsp;</div>
-                              [% ELSE %]
-                              <div class='notificationsDISABLED'>&nbsp;&nbsp;DISABLED&nbsp;&nbsp;</div>
-                              [% END %]
-                            </td>
-                          </tr>
-                          <tr>
-                            <td class='dataVar'>Event Handler:</td>
-                            <td class='dataVal eventHandler'>
-                              [% IF !service.event_handler %]
-                              <div class='eventhandlersENABLED'>&nbsp;&nbsp;UNUSED&nbsp;&nbsp;</div>
-                              [% ELSIF service.event_handler_enabled %]
-                              <div class='eventhandlersENABLED'>&nbsp;&nbsp;ENABLED&nbsp;&nbsp;</div>
-                                [% IF   show_full_commandline == 2 ||
-                                      ( show_full_commandline == 1 && c.check_user_roles('authorized_for_configuration_information') )
-                                %]
-                                <div class='eventhandlersCOMMAND'>([% service.event_handler %])</div>
-                                [% END %]
-                              [% ELSE %]
-                              <div class='eventhandlersDISABLED'>&nbsp;&nbsp;DISABLED&nbsp;&nbsp;</div>
-                              [% END %]
-                            </td>
-                          </tr>
-                          <tr>
-                            <td class='dataVar'>Flap Detection:</td>
-                            <td class='dataVal'>
-                              [% IF service.flap_detection_enabled %]
-                              <div class='flapdetectionENABLED'>&nbsp;&nbsp;ENABLED&nbsp;&nbsp; </div>
-                              [% ELSE %]
-                              <div class='flapdetectionDISABLED'>&nbsp;&nbsp;DISABLED&nbsp;&nbsp; </div>
-                              [% END %]
-                            </td>
-                          </tr>
-                        </table>
-                      </td>
-                    </tr>
-                  </table>
-                </td>
-              </tr>
-            </table>
-          </td>
-          <td align="center" valign="top" class="action_menu">
-            [% IF c.config.use_new_command_box %]
-              [% PROCESS _service_command_box.tt %]
-            [% ELSE %]
-              [% PROCESS _service_command_box_full.tt %]
-            [% END %]
-          </td>
-        </tr>
-
         [% IF pnp_url %]
         <tr>
           <td colspan="2" align="center">
@@ -600,6 +504,25 @@
           </td>
         </tr>
         [% END %]
+                        </table>
+                      </td>
+                    </tr>
+                  </table>
+                </td>
+              </tr>
+
+            </table>
+          </td>
+          <td align="center" valign="top" class="action_menu">
+            [% IF c.config.use_new_command_box %]
+              [% PROCESS _service_command_box.tt %]
+            [% ELSE %]
+              [% PROCESS _service_command_box_full.tt %]
+            [% END %]
+          </td>
+        </tr>
+
+
       </table>
     </div>
 
